paasta_tools/vitesskeyspace_tools.py

```diff
@@ -31,7 +31,7 @@
 log.addHandler(logging.NullHandler())
 
 
-KUBERNETES_NAMESPACE = "paasta-vitessclusters"
+VITESSKEYSPACE_KUBERNETES_NAMESPACE = "paasta-vitessclusters"
```

Comment:
hmm, i think we only ever half-started using KUBERNETES_NAMESPACE for it's intended purpose - iirc, this was meant to be a standard constant so that we didn't need to keep a mapping from workload->namespace

Reply:
since this diff isn't really doing anything special with the renamed constant, i'd probably leave the naming alone to be consistent with the rest of the instance types

Reply:
While this is correct, we'd have to import the whole .*_tools module instead of importing just the variable from the module since the variable name would be the same.
For example,

> from paasta_tools.vitesscell_tools import KUBERNETES_CONFIG
from paasta_tools.vitesskeyspace_tools import KUBERNETES_CONFIG

would create ambiguity for variable value

> from paasta_tools import vitesscell_tools

and referencing the variable with vitesscell_tools.KUBERNETES_CONFIG would eliminate the ambiguity but would require to import the module entirely.

Lmk your thoughts

Reply:
`from paasta_tools.vitesscell_tools import KUBERNETES_NAMESPACE as VITESS_CELL_NAMESPACE`, etc perhaps?